client/src/AppBundle/Controller/Report/ReportController.php

```diff
@@ -19,7 +19,7 @@
 use AppBundle\Service\Client\Internal\ReportApi;
 use AppBundle\Service\Client\Internal\UserApi;
 use AppBundle\Service\Client\RestClient;
-use AppBundle\Service\CsvGeneratorService;
+use AppBundle\Service\Csv\TransactionsCsvGenerator;
```

Comment:
Missing strict_types declaration?

Reply:
I'm leaving this one out as we aren't consistently passing the right values to some arguments that should be ints or strings so would lead to more code changes than this PR concerns.